tfe/data_source_outputs.go

```diff
@@ -159,19 +159,28 @@ type outputData struct {
 
 func (d dataSourceOutputs) readStateOutput(ctx context.Context, tfeClient *tfe.Client, orgName, wsName string) (*stateData, error) {
 	log.Printf("[DEBUG] Reading the Workspace %s in Organization %s", wsName, orgName)
-	opts := &tfe.WorkspaceReadOptions{
-		Include: []tfe.WSIncludeOpt{tfe.WSOutputs},
-	}
-	ws, err := tfeClient.Workspaces.ReadWithOptions(ctx, orgName, wsName, opts)
+	ws, err := tfeClient.Workspaces.Read(ctx, orgName, wsName)
 	if err != nil {
 		return nil, fmt.Errorf("Error reading workspace: %w", err)
 	}
 
+	svOpts := &tfe.StateVersionCurrentOptions{
+		Include: []tfe.StateVersionIncludeOpt{tfe.SVoutputs},
+	}
+	sv, err := tfeClient.StateVersions.ReadCurrentWithOptions(ctx, ws.ID, svOpts)
+	if err != nil {
+		return nil, fmt.Errorf("Error reading state version: %w", err)
+	}
+
+	outputs, err := tfeClient.StateVersions.ListOutputs(ctx, sv.ID, nil)
```

Comment:
Nonblocking but useful feedback: This API reveals sensitive outputs but it's on the list of things to deprecate in the future because of this behavior.

One [alternative possibility](https://github.com/hashicorp/terraform/blob/main/internal/cloud/state.go#L106-L123) is to fetch individual outputs if they are sensitive.

Reply:
Blocking feedback: this endpoint is paginated so we have to consume all the pages.

Reply:
Decided to avoid the API that is deprecated and instead load an individual output if it is sensitive. Thanks for the suggestion!